barretenberg/cpp/src/barretenberg/ecc/curves/bn254/fq.test.cpp

```diff
@@ -526,4 +528,17 @@ TEST(fq, NegAndSelfNeg0CmpRegression)
     a_neg = 0;
     a_neg.self_neg();
     EXPECT_EQ((a == a_neg), true);
+}
+
+// This test shows that ((lo|hi)% modulus) in uint512_t is equivalent to (lo + 2^256 * hi) in field elements so we
+// don't have to use the slow API
```

Comment:
what is the "slow API"? might be useful to make this more explicit so that this comment does not have implied context that future readers might not have

Reply:
Added